pkgs/development/libraries/caliper/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib, stdenv, fetchurl, python3, cmake, adiak, papi
+, withCuda ? false, cudatoolkit
+, withRocm ? false, rocm-runtime
+, withFortran? false, gfortran
+, withOpenmpi? false, openmpi
+, withMpich ? false, mpich
+, libunwindSupport ? true, libunwind
+, libpfmSupport ? true, libpfm
+, elfutilsSupport ? true, elfutils
+, shared ? !stdenv.hostPlatform.isStatic
+}:
+
+# https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/caliper/package.py
+
+let
+   onOffBool = b: if b then "ON" else "OFF";
+   withMPI = (withMpich == true || withOpenmpi == true);
+in
+
+# We can't have both mpis being used (is this the right way to do this?)
+assert (withMpich && !withOpenmpi) || (!withMpich && withOpenmpi) || (!withMpich && !withOpenmpi);
+
+stdenv.mkDerivation rec {
+  pname = "caliper";
+  version = "2.8.0";
+
+  src = fetchurl {
```

Comment:
Use fetchFromGitHub

Reply:
I wasn't sure how this translates, and I don't have my nix setup anymore to test it out, so left as is for now (that I know was working)

Reply:
If you don't have nix anymore, how do you plan to maintain this package?